torch/_inductor/codecache.py

```diff
@@ -1115,7 +1115,7 @@ class CompiledFxGraph:
     mutated_input_idxs: Set[int]
     constants: Dict[str, torch.Tensor]
     torchbind_constants: Dict[str, torch._C.ScriptObject]
-    output_strides: Optional[List[Optional[Tuple[int, ...]]]]
+    output_strides: Optional[List[Optional[Tuple[str, ...]]]]
```

Comment:
Consider documenting the meaning of the string here, either with a type alias or comment